_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "OpenSearch Partner Highlight: Using Cloaked Search to Protect Your Data"
+authors:
+  - zmre
+date:   2022-11-09 1
+categories:
+  - partners
+---
+
+Most companies will tell you that the data they hold is well protected. They'll say it's encrypted “at-rest and
+in-transit,” which means they use HTTPS and disk-level or database-level encryption. Unfortunately, this isn't
+the protection most people think it is.
+
+## Most Encryption Claims Are Smoke and Mirrors
+
+What passes for data protection these days is often wildly overblown on the “protection” front.
+
+HTTPS protects the data for a brief moment in time by warding off eavesdroppers and interlopers, but it
+**doesn't change the availability of the data**. For example, this blog is served over HTTPS, and you are
+probably not authenticated.
+
+“At-rest encryption” is really only useful when someone steals a hard drive. When used by an always-on server,
+the value drops near to zero. That's because the encryption key is only needed when the system starts up and
```

Comment:
```suggestion
the value drops to near zero. That's because the encryption key is only needed when the system starts up, and
```

Reply:
I suggest "...is only needed when the system starts up, after which the protection is transparent...".